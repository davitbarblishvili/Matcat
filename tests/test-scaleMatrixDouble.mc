func main() int {
    matrix a; 
    matrix b; 

    a = [[1,2,3],[4,5,6],[2,10,2]];
    b = 2.5 * a;

    printm(b);

    return 0;

}