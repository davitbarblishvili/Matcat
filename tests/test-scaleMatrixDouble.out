(
[2.50 5.00 7.50 ]
[10.00 12.50 15.00 ]
[5.00 25.00 5.00 ]
)